ledger/participant-integration-api/src/main/scala/platform/apiserver/services/ApiVersionService.scala

```diff
@@ -74,7 +75,9 @@ private[apiserver] final class ApiVersionService private (
           optionalLedgerId = Some(ExperimentalOptionalLedgerId()),
           contractIds = Some(ledgerFeatures.contractIdFeatures),
           committerEventLog = Some(ledgerFeatures.committerEventLog),
-          explicitDisclosure = None, // TODO[ED]: Wire-up with participant configuration flag
+          explicitDisclosure = None, // TODO[ED]: Wire-up with participant configuration flag,
+          userAndPartyManagementExtensionsForHub =
+            Some(ExperimentalUserAndPartyManagementExtensionsForHub(supported = true)),
```

Comment:
Are you going to add a flag for this?

Reply:
No, I don't intend to add a flag.

I mulled over whether to have a 'supported: bool' field but I'm still not sure. It's not probably of not a great importance as I exposed it as an experimental feature.